games-rpg/freedroidrpg/freedroidrpg-0.16.1.recipe

```diff
@@ -102,19 +102,6 @@ PATCH()
 
 BUILD()
 {
-	echo "=== libtoolize"
-	libtoolize --force --copy --install
-	echo "=== aclocal"
-	aclocal
-	echo "=== automake"
-	automake --add-missing
-	echo "=== autoreconf"
-	autoreconf
-	echo "=== autoconf"
-	autoconf
-	echo "=== automake"
-	automake
-	runConfigure ./configure --disable-sdltest --with-embedded-lua
```

Comment:
are you sure running configure is not necessary?

Reply:
hmm probably removed the line to go straight to make for checking :)

Reply:
Fixed :)